src/table/ITable.ts

```diff
@@ -99,7 +112,12 @@ export interface ITable extends IDataType {
   queryView(name: string, args: IQueryArgs): ITable;
 
   /**
-   * TODO: document
+   * Creates a new view on this table specified by the given range of IDs. A view implements the ITable interface yet is
+   * still backed by the data from the original table.
+   *
+   * When passing a single (1D range) the range applies to the rows.
+   * When passing a 2D range, the row-range is in the first, the col-range in the
```

Comment:
it doesn't support 2D ranges, since columsn doesn't have unique ids that can be mapped